homeassistant/components/cover/knx.py

```diff
@@ -50,20 +58,42 @@ def __init__(self, hass, config):
         KNXMultiAddressDevice.__init__(
             self, hass, config,
             ['updown', 'stop'],  # required
-            optional=['setposition', 'getposition']
+            optional=['setposition', 'getposition',
+                      'getangle', 'setangle']
         )
         self._device_class = config.config.get(CONF_DEVICE_CLASS)
         self._invert_position = config.config.get(CONF_INVERT_POSITION)
+        self._invert_angle = config.config.get(CONF_INVERT_ANGLE)
         self._hass = hass
         self._current_pos = None
         self._target_pos = None
+        self._current_tilt = None
+        self._target_tilt = None
+        self._supported_features = SUPPORT_OPEN | SUPPORT_CLOSE | \
+            SUPPORT_SET_POSITION | SUPPORT_STOP
+
+        # Tilt is only supported, if there is a angle get and set address
+        if (config.config.get(CONF_SETANGLE_ADDRESS) is not None) and \
```

Comment:
Remove also `_LOGGER.debug("%s: Tilt not supported", self.name)` that is not needed. It is loaded or not, you have debug output already there.